Sonarqube-intermediate.html

```
@@ -0,0 +1,3681 @@
+                }
+                .content {
+                    padding: 70px 20px 100px;
+                }
+                header.hero h1 {
+                    font-size: 26px;
+                }
+            }
+        </style>
+    </head>
+    <body>
+        <button class="menu-toggle" id="menuToggle" aria-label="メニューを開く">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)